kyaml/openapi/openapi.go

```diff
@@ -593,14 +605,10 @@ func initSchema() {
 	globalSchema.schemaInit = true
 
 	if customSchema != nil {
-		err := parse(customSchema)
+		err := parse(customSchema, "json")
 		if err != nil {
 			panic("invalid schema file")
 		}
-		if err = parse(kustomizationapi.MustAsset(kustomizationAPIAssetName)); err != nil {
-			// this should never happen
-			panic(err)
-		}
```

Comment:
Why don't we need this anymore?

Reply:
We do, I just moved it out to `initSchema` because it's needed whether we have a builtin or custom schema.

Reply:
Need to remove the `return` below at L604 then! (and I guess this must not be tested)

Reply:
done! Thanks for the catch